backend/siarnaq/api/teams/views.py

```diff
@@ -173,3 +174,35 @@ def avatar(self, request, pk=None, *, episode_id):
             titan.upload_image(avatar, profile.get_avatar_path())
 
         return Response(None, status=status.HTTP_204_NO_CONTENT)
+
+    @action(
+        detail=False,
+        methods=["post"],
+        serializer_class=None,
+        permission_classes=(IsAuthenticated, IsEpisodeAvailable),
+    )
+    def record(self, request, pk=None, *, episode_id):
+        """Retrieve the win/loss record of a team"""
+        match_participations = MatchParticipant.objects.filter(
+            team_id=request.data["id"]
+        )
```

Comment:
this filter is not functioning as i expect. The goal is to grab every Match particpant obj that is related to the team id, but it never seems to match.

Reply:
Here's an example of filtering matches for a particular team: https://github.com/battlecode/galaxy/blob/0e8754516fcb667cf7f2fee92864d9f15e14d7a8/backend/siarnaq/api/compete/views.py#L304